benchmark/_cli.js

```diff
@@ -82,17 +82,35 @@ CLI.prototype.abort = function(msg) {
 
 CLI.prototype.benchmarks = function() {
   const paths = [];
-  const filter = this.optional.filter || false;
 
   for (const category of this.items) {
     if (benchmarks[category] === undefined)
       continue;
     for (const scripts of benchmarks[category]) {
-      if (filter && scripts.lastIndexOf(filter) === -1) continue;
+      if (this.shouldSkip(scripts)) continue;
 
       paths.push(path.join(category, scripts));
     }
   }
 
   return paths;
 };
+
+CLI.prototype.shouldSkip = function(scripts) {
+  let matchPattern = false;
+  const filters = this.optional.filter || [];
+  for (let filter of filters) {
+    const negatePattern = filter.startsWith('-');
```

Comment:
I am somewhat conflicted on this idea - `-` can and is used in benchmark file names, so this makes it impossible to include files that contains `-foo` but exclude files that are e.g. `barfoo`, which seems to be an entirely plausible use case.

Can we use a separate flag for this? e.g. `--exclude`?

Reply:
I'm fine changing it to `--exclude`.

Reply:
Done